src/sas/qtgui/Perspectives/Fitting/media/fitting_help.rst

```diff
@@ -650,6 +650,41 @@ The DREAM optimiser is the most sophisticated, but may not necessarily be the be
 option for fitting simple models. If uncertain, try the Levenberg-Marquardt optimiser
 initially.
 
+Some model parameters, for example, radii/lengths or orientation angles can be
+polydisperse; i.e. they can have a distribution of possible values. Polydisperse
+parameters are defined as such when the model is coded, and can be activated by
+clicking the *Polydispersity* checkbox on the *Fit Page*.
+
+.. image:: enable_pd.png
+
+Clicking on the *Polydispersity* tab then provides access to these polydisperse
+parameters and allows the type (i.e. the *function* to be used) and 'width'
+(the *PD[ratio]*) to be adjusted. If necessary the 'step size' (*Npts*) and
+'range' (*Nsigs*) of the function can also be adjusted.
+
+.. image:: pd_tab.png
+
+For more information, see the descriptions of :ref:`polydispersityhelp` . In
+particular, pay attention to the Suggested Applications and Usage Notes therein.
+Note that SasView defaults to Gaussian distributions, but these will not always
+be the best choice. Also, the definition of *PD[ratio]* varies depending on
+the chosen distribution!
```

Comment:
PD for orientation is absolute; PD for volume parameters are relative to the current value. It doesn't depend on the chosen distribution, though the meaning of center and width does (first and second moments was a good analytic choice, but it sure was confusing for rectangular distributions).

Reply:
Thanks @pkienzle . Have changed it to read now:

![image](https://user-images.githubusercontent.com/10938679/139681166-29a82cce-e98a-4b3a-8563-ba53f2a9d16b.png)